axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/PurchaseOrderInvoiceServiceImpl.java

```diff
@@ -126,12 +126,13 @@ public Invoice generate() throws AxelorException {
 
     for (PurchaseOrderLine purchaseOrderLine : purchaseOrderLineList) {
 
-      processPurchaseOrderLine(invoice, invoiceLineList, purchaseOrderLine);
+      Beans.get(PurchaseOrderInvoiceService.class)
```

Comment:
Please do not inject a service from itself, you do not have to inject again the service to get overridden methods